api/envoy/type/percent.proto

```diff
@@ -31,15 +31,25 @@ message FractionalPercent {
     // **Example**: 1/100 = 1%.
     HUNDRED = 0;
 
+    // 1,000.
+    //
+    // **Example**: 1/1000 = 0.1%.
+    THOUSAND = 1;
+
     // 10,000.
     //
     // **Example**: 1/10000 = 0.01%.
-    TEN_THOUSAND = 1;
```

Comment:
These can't be changed. You will need to add new ones at the end. Are they really necessary though? Seems unnecessary to me with TEN_THOUSAND and MILLION. I would revert these changes personally.

Reply:
Addressed in f105661.